packages/netlify-cms-widget-relation/src/RelationControl.js

```diff
@@ -128,24 +155,26 @@ export default class RelationControl extends React.Component {
   parseHitOptions = hits => {
     const { field } = this.props;
     const valueField = field.get('valueField');
-    const displayField = field.get('displayFields') || field.get('valueField');
+    const displayField = field.get('displayFields') || List(field.get('valueField'));
```

Comment:
Makes sure `displayField` is always a list to avoid checking it later like https://github.com/netlify/netlify-cms/pull/3754/files#diff-1c6e2a499213935b7c1b04dbb4e1ca1cL135